src/Storages/System/StorageSystemCompletions.cpp

```diff
@@ -89,7 +94,19 @@ void fillDataWithTableColumns(
     if (table_lock == nullptr)
         return; // table was dropped while acquiring the lock
 
-    const auto & snapshot = table->getInMemoryMetadataPtr();
+    StorageMetadataPtr snapshot;
+    /// Can throw UNKNOWN_DATABASE/UNKNOWN_TABLE in case of Alias table
+    try
+    {
+        snapshot = table->getInMemoryMetadataPtr();
+    }
+    catch (const Exception & e)
+    {
+        if (table->getName() != "Alias" || e.code() == ErrorCodes::UNKNOWN_DATABASE || e.code() == ErrorCodes::UNKNOWN_TABLE)
+            return;
+        throw;
```

Comment:
This looks wrong, let's use the same form as in system.columns:

```suggestion
        if (table->getName() != "Alias" || (e.code() != ErrorCodes::UNKNOWN_DATABASE && e.code() != ErrorCodes::UNKNOWN_TABLE))
                throw;
        tryLogCurrentException(getLogger("SystemCompletions"), fmt::format("Cannot find target database/table for {}", table->getStorageID().getNameForLogs()), LogsLevel::debug);
        return;
```